RecoParticleFlow/PFTracking/src/PFDisplacedVertexFinder.cc

```diff
@@ -734,6 +732,17 @@ PFDisplacedVertexFinder::getTransvDiff(const GlobalPoint& Ref, const GlobalPoint
 
 }
 
+std::pair<float,float>
```

Comment:
Based on jenkins tests in https://github.com/cms-sw/cmssw/pull/20997	ea15459
there are no differences. So, getTransvDiff and getLongDiff can be removed.